include/rive/artboard.hpp

```diff
@@ -128,6 +133,15 @@ namespace rive {
         StateMachine* firstStateMachine() const;
         StateMachine* stateMachine(std::string name) const;
         StateMachine* stateMachine(size_t index) const;
+
+        /// Make an instance of a state machine given the name of the source state machine in this
+        /// artboard. The StateMachineInstance must be explictly deleted when no longer needed.
+        StateMachineInstance* stateMachineInstance(std::string name);
```

Comment:
Logically, these already were added to ArtboardInstance -- do we need them here?

Reply:
ArtboardInstance should only have a reference to the source StateMachine when it is first instanced from the source Artboard.

Reply:
I think this is a byproduct of not having merged your latest changes in this branch, working on that now.